test/Integration/gemm.mlir

```diff
@@ -1,4 +1,4 @@
-// RUN: tpp-opt %s -empty-tensor-to-alloc-tensor -one-shot-bufferize="bufferize-function-boundaries allow-return-allocs function-boundary-type-conversion=identity-layout-map" -drop-equivalent-buffer-results -finalizing-bufferize --linalg-ext-to-loops --convert-check-to-loops --convert-vnni-to-tpp --convert-linalg-to-tpp -canonicalize -convert-tpp-to-xsmm -convert-xsmm-to-func -canonicalize -convert-linalg-to-loops -convert-vector-to-scf -convert-scf-to-cf -expand-strided-metadata -lower-affine  -arith-expand -reconcile-unrealized-casts |\
+// RUN: tpp-opt %s -empty-tensor-to-alloc-tensor -one-shot-bufferize="bufferize-function-boundaries allow-return-allocs function-boundary-type-conversion=identity-layout-map" -drop-equivalent-buffer-results -finalizing-bufferize -buffer-results-to-out-params -buffer-deallocation  --linalg-ext-to-loops --convert-check-to-loops --convert-vnni-to-tpp --convert-linalg-to-tpp -canonicalize -convert-tpp-to-xsmm -convert-xsmm-to-func -canonicalize -convert-linalg-to-loops -convert-vector-to-scf -convert-scf-to-cf -expand-strided-metadata -lower-affine  -arith-expand -reconcile-unrealized-casts |\
```

Comment:
How does that compare with the default pipeline? @adam-smnk

Reply:
It looks pretty much the same.
`-default-tpp-passes -buffer-results-to-out-params -buffer-deallocation -lower-affine -arith-expand` should also work.

Reply:
We probably should move all integration tests to the default pipeline soon